src/components/Backtester/reports/HistoricalReport.js

```diff
@@ -47,6 +61,32 @@ const HistoricalReport = (opts, results, backtestData, backtestOptions, t, setti
     ]
   ))
 
+  const onTradeExportClick = () => {
```

Comment:
could you please move this fn definition to helper file, so this comp can only focus on single-responsibility of rendering UI ?